Simplenote/src/main/java/com/automattic/simplenote/AddTagActivity.java

```diff
@@ -131,6 +131,12 @@ public void afterTextChanged(Editable s) {
             return;
         }
 
+        if (tag.contains(",")) {
```

Comment:
case-in-point about the constant use of `COMMA` - we're not using it here and nothing suffers from inlining this literal.